### ISSUE TITLE
Success Criterion 3.1.7 Plain Language (Minimum) - evaluating the accessibility of language

### ISSUE THREAD

**Issue by Eliisa** [w3c/wcag21#210]
It’s very good that this guideline sets requirements for language used in websites and in the same time says something about language’s relation with accessibility and understandability.  However, in this form criterion may raise more questions than it answers.

First question is why plain text is required only for navigational elements, user interfaces, and instructions? In Issue 30 concerning this plain language (minimum) is said: “Clear language for all content is an important accessibility principle.” 

So why this exclusion?  Body text (content) of a page or a site contains all the information (or entertainment) that the user is looking for. Getting to the information is vital and often is forgotten but still, there’s no need or use for such sites that provide easy access by accessible navigation with content written in difficult, incomprehensible language.  

Furthermore it would be advisable to try to determine more specifically the degree of difficulty that is required  for accessible language (in spectrum of occupational jargon – plain language – easy-to-read) and what situations / fields in website should be in the easiest possible to truly improve accessibility.

When focusing on the list/bullets in this criterion, there are few issues concerning those chosen five bullets and their ability to make any website more accessible. Here are some notes on this subject:

- It is good, that there’s a clear aim to cover more than just one level of language (e. g. vocabulary).  
- Languages are different in terms of how example tense, voice and grammatical cases are constructed. Thus there are different things in every language that make it easy or difficult to read.  Criterion does not address different languages which is good in the sense that now it can be adapted to all but then this might weaken the possibilities to really evaluate the actual accessibility of given website.  

- **Simple tense and voice**: It seems that when speaking of loose sentences active voice seems always easier, but in reality choices are always made in some context. In given context passive voice can be easily easier thus making this bullet very hard to be evaluated without deeper study of the text. In exceptions it’s said that “Other voices or tenses may be used when it has been shown to be easier to understand, friendlier, or appropriate” but how things like these can be measured or evaluated objectively and reliably?
- **Simple, clear, and common words**: To some extent this sounds reasonable, but 1500 words is a lot, especially when you can have a new set of common words with every new topic. It is questionable whether this sort of limitation helps understandability (and accessibility). 
- **Double negatives:** This concerns only certain languages. 
- **Concrete language:** Mentioned automatic replacement sounds may be too good to be true: is it really possible to do so reliably? Automatic replacements could do serious harm to the text, it’s coherence and understandability, in other words work against accessibility. 
- **Instructions**: What identification means in this bullet?

-  **Exceptions**: It is good that guideline sees the value and meaning of context and gives an opportunity to make choices that serve the text better. However, it raises even more questions about the qualifications of a evaluator and his/her capability of figuring out whether or not valued text is accessible by this criterion. 

These bullet points and especially exceptions require specific knowledge of language in question and deep understanding of what makes the language accessible. This is a big problem, and if these kind of decisions are made lightly, without enough knowledge of linguistics or subjectively, it questions the whole meaning of this criterion.

Making language easier is never done by switching one word to another. Accessibility (especially in congnitive sence) should be viewed as a larger phenomenon concerning all aspects of texts: 

1. the text as a whole  e. g. it’s chosen viewpoint, style and the way it addresses the reader. 
2. chosen vocabulary and explanations for difficult words (which can’t be avoided).
3. The way writer constructs clauses and sentences. 

Language and understandability are not easily measured automatically and thus there has been little said about this crucial side of accessing information.  Still, leaving it out is even more problematic and endangers the whole idea of accessible websites. 

Eliisa Uotila, The Finnish Centre for Easy to read

**Comment by lseeman** [comment]
refrencing this comment  #30

**Comment by awkawk** [comment]
Thanks for the comment - the Working Group has not reached consensus on this proposed SC so it is deferred for future consideration.
